r/restic/restic_ubi8.4.sh

```diff
@@ -0,0 +1,85 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : restic
+# Version       : v0.12.1
+# Source repo   : https://github.com/restic/restic/
+# Tested on     : rhel UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer's  : Santosh Magdum <santosh.magdum@us.ibm.com>
+#                 Priya Seth <priya.seth@us.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+PACKAGE_NAME=restic
+PACKAGE_PATH=github.com/restic/restic
```

Comment:
looks like we are diverging from the template and the variables it sets here https://github.com/ppc64le/build-scripts/blob/66d9be10cc51eecd1859e81cc3f27d17cd4fca25/templates/build_script_go.sh#L20 , any specific reason for that?

Reply:
The PAKAGE_PATH is used to perform go operations and cd to the directory.

cd $(ls -d $GOPATH/pkg/mod/$PACKAGE_PATH@$PACKAGE_VERSION)